pipelines/full-build.yaml

```
@@ -660,15 +272,40 @@ stages:
             export BUILT_AS=cccstemp.azurecr.io/${CONTAINER_NAME}:$TAG
             docker pull $BUILT_AS
 
-            for IMAGE in "cccs/" "docker.pkg.github.com/cybercentrecanada/assemblyline/" "uchimera.azurecr.io/cccs/"
+            # Publish to public container registries
+            for IMAGE in "cccs/" "docker.pkg.github.com/cybercentrecanada/assemblyline/"
             do
               docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$TAG
               docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$BUILD_TYPE
               docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$SERIES
               docker push ${IMAGE}${CONTAINER_NAME} --all-tags
             done
+
+            # Publish to private container registry (append labelling)
+            docker build -t $BUILT_AS -<<EOF
+            FROM $BUILT_AS
+            LABEL classification=UNCLASSIFIED
+            EOF
+
+            IMAGE=uchimera.azurecr.io/cccs/
+            docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$TAG
+            docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$BUILD_TYPE
+            docker tag $BUILT_AS ${IMAGE}${CONTAINER_NAME}:$SERIES
+            docker push ${IMAGE}${CONTAINER_NAME} --all-tags
+
           displayName: Deploy base Docker Hub
           retryCountOnTaskFailure: 3
+
+        - script: |
+            echo "##vso[task.setvariable variable=github_repo;isreadonly=true]${DIRECTORY:-${CONTAINER_NAME}}"
```

```suggestion
        - script: |
            echo "##vso[task.setvariable variable=github_repo;isreadonly=true]${DIRECTORY:-${CONTAINER_NAME}}"
          displayName: Configure Github variable
```
Should this be part of publish-description.yaml?